spec/fixtures/vcr_cassettes/customers/create/valid.yml

```diff
@@ -0,0 +1,57 @@
+---
+http_interactions:
+  - request:
+      method: post
+      uri: https://connect.squareupsandbox.com/v2/customers
+      body:
+        encoding: UTF-8
+        string: '{"given_name":"John","family_name":"Doe","email_address":"email1@example.com","address":{"address_line_1":"A
+        Different Road","address_line_2":"A Different Road","locality":"Herndon","postal_code":"10002","country":"US"},"phone_number":"555-555-0199","reference_id":"1"}'
+      headers:
+        Accept:
+          - application/json
+        Content-Type:
+          - application/json; charset=utf-8
+        Authorization:
+          - Bearer EAAAEJ2wQT1qUQIvMGlImkhphBakgTk-r8rhyznGQ_5IvSKFpKk4vmMexZRGAIgR
```

Comment:
Please filter out the sensitive data with this [code](https://github.com/solidusio-contrib/solidus_paypal_marketplace/blob/main/spec/support/vcr.rb#L15-L17)